errors/template.go

```diff
@@ -41,11 +42,15 @@ type Template struct {
 }
 
 func NewTemplateFromFile(path string) (*Template, error) {
-	tplFile, err := ioutil.ReadFile(path)
+	absPath, err := filepath.Abs(path)
```

Comment:
Wozu diese Änderung? Der Code funktioniert auch ohne.

Reply:
The absolute path was created on the callee side before and was a self-made one. Using the build in method abstracts OS related path resolving.